README.md

```diff
@@ -1,2 +1,69 @@
 # Online microbiome data analysis
 
+
+***Involve and empower citizens to contribute to science***
+
+
+## Welcome!
+
+Welcome! :tada: Willkommen! :balloon: Grüezi! :confetti_ball: Bienvenue! :balloon::balloon::balloon:
+
+Thank you for visiting the online microbiome data analysis project repository.
+
+This document (the README file) is a hub to give you some information about the project. Jump straight to one of the sections below, or just scroll down to find out more.
+
+* [What are we doing? (And why?)](#what-are-we-doing)
+* [Who are we?](#who-are-we)
+* [What do we need?](#what-do-we-need)
+* [How can you get involved?](#get-involved)
+* [Get in touch](#contact-us)
+
+## What are we doing?
+
+### The problem
+
+Especially in the last years public media is using words like sequencing, DNA, RNA, mutations, and variants, without easy to understand scientific explanations. This leaves the audience not more informed and sometimes even more confused.
```

Comment:
Theses key words do not necessarily help informing the audience and sometimes leads to even more confusion.

Reply:
changed. :+1: